test/resources/ostree.resource

```diff
@@ -90,3 +90,18 @@ Wait For Transaction To End
 
     Wait Until Keyword Succeeds    2m    15s
     ...    No Transaction In Progress
+
+Journal Should Have Information About Failed Version Comparison
+    [Documentation]    Assert that unhealthy deployment's journal contains
+    ...    information about failed version compatibility check
+
+    FOR    ${boot}    IN RANGE    -3    0
```

Comment:
I see you're just moving this, but the loop looks a little odd. Should there be a sleep somewhere? Or should this be a "wait until keyword succeeds" loop?

Reply:
Not sure, why?
I need to execute journalctl 3 times with different `--boot` values: `-3`, `-2, and `-1`. This happens when system rolls back and it's only `journalctl | grep`